### PR TITLE
Feat side bar/#114

### PR DESCRIPTION
**Pull request by flaviafreirec**
Issue: #114 

<details open> 
  <summary>
    <b>Feature</b>
  </summary>

- Create sidebar and header section.
</details>

<details open> 
  <summary>
    <b>Bugfix</b>
  </summary>

- **Description**
Adjust main container borders and paddings

- **Cause**
The borders and paddings was just for testing, and were conflicting with our new components

- **Solution**
Remove borders and adjust paddings
</details>

<details open> 
  <summary>
    <b>Changelog</b>
  </summary>

- [Add new `header` section](https://github.com/Alecell/octopost/pull/136/commits/0b249464b3c842f7b270053f9472e2325eb1ac0f)
- [Add new `sidebar` section](https://github.com/Alecell/octopost/pull/136/commits/41fc66c5dd92b5ed3be8edcf81a8bddab4972021)
- [Adjust sizes of `home` containers](https://github.com/Alecell/octopost/pull/136/commits/10303c7106b283d0bf42579dc82fe7d5ce9bfd38)

</details>

<details open> 
  <summary>
    <b>Visual evidences :framed_picture:</b>
  </summary>

page

</details>

<details open> 
  <summary>
    <b>Checklist</b>
  </summary>

  - [x] Issue linked
  - [x] Build working correctly
  - [ ] Tests created
</details>

### CODE REVIEW

File: src/pages/home/components/Sidebar/Sidebar.module.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+* {
+  box-sizing: border-box;
+}
```
Comment (aalmeida00) [inline src/pages/home/components/Sidebar/Sidebar.module.scss:3]: Isso daqui nao deveria estar no base.scss e me pergunto além se isso nao deveria funcionar no normalize.css

**Review by aalmeida00 — CHANGES_REQUESTED** [review]

File: src/pages/home/components/Sidebar/Sidebar.module.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+.sidebar {
+  width: 100%;
+
+  font-size: 1.6rem;
+
+  border: 1px solid #cbc4cf;
+
+  background-color: #000;
+
+  background-color: #f5eff4;
```
Comment (Luiginicoletti) [inline src/pages/home/components/Sidebar/Sidebar.module.scss:14]:
```suggestion
  background-color: #000;

  background-color: #f5eff4;
```

Tem dois background-color.

**Comment by kingjotaro** [comment]
A senhorita Flavia não assinou o PR

**Review by aalmeida00 — APPROVED** [review]

**Comment by flaviafreirec** [comment]
adicionei o ladle stories para o header e sidebar hj c um novo commit

**Review by aalmeida00 — APPROVED** [review]

**Pull request merged**